### DISCUSSION TITLE
Replace "No user logged" with OP username.

### DISCUSSION THREAD

**Discussion by SuperTrollMan** [OctoFarm/OctoFarm#1100] [Implemented]
I can see it could be beneficial on the History tab, to list the OP user who initiated the print.
If for example your silcer talks direct to the OP instance (uploads and prints straight to the printer) OF lists the user as "No user logged", which doesn't help identify who requested the print.

It could work as follows:
If a print is initiated from OF, the user is logged with the OF users name.
If a print is initiated from OP (or from a 3rd party tool to OP), the user could be pulled from OP and listed as "<OP Username>(OctoPrint)" or similar, which would let users know it was triggered outside of OF and who triggered it.

![Screenshot 2022-06-19 104436](https://user-images.githubusercontent.com/95956617/174463716-2b0b518b-79d7-4caa-8fef-87e64fa9eb3a.png)

**Comment by NotExpectedYet** [comment]
Yeah currently OF expects everything done OF side but can understand the requirement there. That's a handy fall back to have. Seems I already log it too. 

![image](https://user-images.githubusercontent.com/61137988/174464914-9d4f18d0-ac1f-4ce9-8f08-662b4d0c2e3d.png)

**Comment by NotExpectedYet** [comment]
![image](https://user-images.githubusercontent.com/61137988/174478437-7aa5c0dc-81ae-4a5a-b2bd-30768959164b.png)

Will be in next release
